tests/Fixtures/LoadsFixtures.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace Tests\Fixtures;
+
+
+trait LoadsFixtures
+{
+    /**
+     * Loads fixtures for use in data providers.
+     *
+     * @param string $directory
+     * @param int $limit
+     * @return iterable
+     */
+    private function fixtureData(string $directory, int $limit): iterable
```

Comment:
Can we determine the `limit` from examining the directory?

Reply:
I did consider globbing the directory, will see if I can simplify.

Reply:
Done

Reply:
Great solution!